vlc/rootfs/etc/cont-init.d/ingress.sh

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Create VLC secret
+# ==============================================================================
+
+# Generate password
+if bashio::fs.file_exists /data/ingress-secret; then
+    bashio::exit.ok
```

Comment:
This means it will never generate an ingress config on the second run. This add-on can only run once

Reply:
I patterned it after the way you were handling the telnet-secret; is there any harm in generating an ingress secret one time and maintaining the same secret for future runs? That's how it was before when the password was a static value of ":INGRESS".

I don't mind making it generate and use a new ingress secret on every container restart, but it seems counterproductive to me.

Reply:
What do you think will be happening with the code after this?